ceph-build-next/build/build_deb

```diff
@@ -11,11 +11,11 @@ vers=`cat ./dist/version`
 distro=`python -c "exec 'import platform; print platform.linux_distribution()[0].lower()'"`
 chacra_endpoint="ceph/${chacra_ref}/${distro}/{$dist}/${ARCH}"
```

Comment:
for `exists` it needs the full URL because you can check binaries or repos, while `chacractl binary` requires only the url from the project name forwards